cli/core/status.go

```diff
@@ -75,31 +75,52 @@ func sumActiveValidatorBalancesGwei(allValidators []ValidatorWithIndex, allBalan
 	return sumGwei
 }
 
+func sumRestakedBalancesGwei(eth *ethclient.Client, eigenpodAddress string, activeValidators []ValidatorWithIndex) (phase0.Gwei, error) {
+	var sumGwei phase0.Gwei = 0
+
+	validatorInfos, err := GetOnchainValidatorInfo(eth, eigenpodAddress, activeValidators)
```

Comment:
i wonder if we already have this, because we did `SelectActiveValidators` and already fetched the info

Reply:
i can clean this up